product-features/liquidity-providers/how-to.md

```diff
@@ -0,0 +1,223 @@
+---
+description: >-
+  In these two guides, you'll find the required steps to provide liquidity and
+  to withdraw liquidity from ALEX DEX pools.
+---
+
+# How to add/remove liquidity
+
+When **adding liquidity**, you will deposit an equivalent value of both tokens into the pool. In return, you'll receive LP tokens, which represent your share of that specific liquidity pool.
+
+When **removing liquidity**, you will transfer your LP tokens back to the ALEX smart contract and receive your original funds plus any fees accrued while holding those LP tokens.
+
+## :palm\_up\_hand: :moneybag: Adding Liquidity
+
+### Step 1
+
+Go to [https://app.alexlab.co/](https://app.alexlab.co/) and click on navbar's Swap -> Pool tab.
+
+<figure><img src="../../.gitbook/assets/liquidity-providers/adding-liquidity-1-pool-tab.png" alt="" width="375"><figcaption></figcaption></figure>
+
+### Step 2
+
+All available pools will be displayed including information such as:
+
+* **Trading Pair:** The token pair that constitute liquidity pools to which you can add liquidity.
+* **Liquidity:** The total liquidity in the pool, expressed in USD value.
+* **Volume:** The trading volume between the token pair over the last 7 days.
```

Comment:
The trading volume between the token pair over the last 7 days/24 Hours (by hovering on the trading volume).

Reply:
Done

Reply:
Done. Please resolve conversation or comment if any other change requested.